recipes/codechecker/meta.yaml

```
@@ -0,0 +1,60 @@
+{% set name = "codechecker" %}
+{% set version = "6.18.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: b49d9e10b6d53fd95b28db353c19f69e907188d45b6aeff8a5940fee35509534
+
+build:
+  number: 0
+  entry_points:
+    - CodeChecker = codechecker_common.cli:main
+    - merge-clang-extdef-mappings = codechecker_merge_clang_extdef_mappings.cli:main
+    - post-process-stats = codechecker_statistics_collector.cli:main
+    - report-converter = codechecker_report_converter.cli:main
+    - tu_collector = tu_collector.tu_collector:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - python
+    - {{ compiler('cxx') }}
```

Only need `python` listed under the `host` section:
```suggestion
  build:
    - {{ compiler('cxx') }}
```